app/harvesters/hal/hal_references_converter.py

```diff
@@ -71,6 +74,17 @@ def _identifiers(self, raw_data):
             else:
                 yield ReferenceIdentifier(type=field, value=raw_data[field])
 
+    def _date(self, date):
+        # Check if is a valid ISO 8601 date
+        try:
+            if date is None:
+                return None
+            return isodate.parse_datetime(date).replace(tzinfo=None)
+        except isodate.ISO8601Error as error:
+            raise UnexpectedFormatException(
```

Comment:
This will cancel the recording of the whole publication if the date format is erroneous. Perhaps logging may be enough.